src/prop.ts

```diff
@@ -58,6 +58,7 @@ export interface PropOptions extends BasePropOptions {
   ref?: any;
   /** Take the Path and try to resolve it to a Model */
   refPath?: string;
+  refType?: 'number' | 'string' | 'Buffer' | 'ObjectID';
```

Comment:
change to lower case, just to keep everything normal
```suggestion
  refType?: 'number' | 'string' | 'buffer' | 'objectid;
```